### PR TITLE
Metatags helper

### PR DESCRIPTION
**Pull request by cvalarida**
## Description
Added a helper function to create the array of metatags so we didn't need to re-do it every time.

**Note:** This may mean that the metatags are in a different order than expected in some cases, but I have a hard time imagining that's going to be a problem.

## Testing done
Made sure the existing unit tests still passed. I did have to add a few metatags to `node-page` since we're apparently supplied with more than we were getting in `pages.json`. (Which...I'm not sure if that should be concerning or not...)

## Screenshots
N/A

## Acceptance criteria
- [x] The unit tests still pass
- [x] The helper function builds an array of metatag objects with either the `__typename` (default) or `type`

### CODE REVIEW

File: src/site/stages/build/process-cms-exports/tests/transformed-entities/node-page.json [hunk 1]
```diff
@@ -49,34 +49,49 @@
   "entityPublished": true,
   "entityMetaTags": [
     {
-      "type": "MetaValue",
       "key": "title",
+      "type": "MetaValue",
```
Comment (cvalarida) [inline src/site/stages/build/process-cms-exports/tests/transformed-entities/node-page.json]: There were some new metatags, so I got lazy and copied all of them from the output. This apparently meant these properties are now alphabetized. :man_shrugging:

File: src/site/stages/build/process-cms-exports/transformers/node-event.js [hunk 2]
```diff
@@ -17,84 +18,49 @@ function fakeUtc(timeString) {
...
-    ),
-    fieldFacilityLocation: entity.fieldFacilityLocation[0] || null,
-    fieldLink: createLink(entity.fieldLink, ['url']),
-    fieldLocationHumanreadable: getDrupalValue(
-      entity.fieldLocationHumanreadable,
-    ),
-    fieldMedia: entity.fieldMedia[0] || null,
-  };
-};
+const transform = entity => ({
```
Comment (cvalarida) [inline src/site/stages/build/process-cms-exports/transformers/node-event.js]: This looks like a bigger change than it was.
1. I removed `createMetaTag`
2. I used `createMetaTagArray`
3. I changed `entity => { const metaTags = entity.metatag.value; return { ... } }` to `entity => ({ ... })`

**Review by cvalarida — COMMENTED** [review]
It looks like a lot bigger change than it was. :man_shrugging:

File: src/site/stages/build/process-cms-exports/transformers/helpers.js [hunk 3]
```diff
@@ -122,4 +113,35 @@ module.exports = {
...
+      createMetaTag('MetaValue', 'twitter:title', metaTags.twitter_cards_title),
+      createMetaTag('MetaValue', 'twitter:site', metaTags.twitter_cards_site),
+      createMetaTag('MetaProperty', 'og:title', metaTags.og_title),
+      createMetaTag('MetaProperty', 'og:description', metaTags.og_description),
+      createMetaTag(
+        'MetaProperty',
+        'og:image:height',
+        metaTags.og_image_height,
+      ),
+    ].filter(t => t.value);
```
Comment (bkjohnson) [inline src/site/stages/build/process-cms-exports/transformers/helpers.js]: My big concern was having metatags for missing data, so I'm glad that this is here.

**Review by bkjohnson — APPROVED** [review]
Looks good :+1:

**Pull request merged**